react-components/src/tools/EditTool.js

```diff
@@ -167,12 +202,46 @@ function EditToolForm(props) {
                     </FormSelectField>
                 )}
             />
+            {isAdmin && (
+                <Field
+                    name="interactive"
```

Comment:
Since this flag should always be set to `true` when the tool type is `interactive`, maybe we can have the [ToolAdminPresenterImpl](https://github.com/cyverse-de/ui/blob/a8b15fe363e8ece4b32e798d0d7576cd53f9de0b/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/toolAdmin/presenter/ToolAdminPresenterImpl.java#L224-L228) auto-set this field, instead of forcing the admin to remember to set or unset this for `interactive` types. We do this for the (non-admin) DE form already:
https://github.com/cyverse-de/ui/blob/4fbf931cbf72bcb69c71379e2c30b4f088011de7/de-lib/src/main/java/org/iplantc/de/tools/client/presenter/ManageToolsViewPresenter.java#L204-L222
Same for the `skip_tmp_mount` flag below.

Maybe we can also auto-set the `network_mode` to `bridge` for `interactive` tools in the admin form, but we should still allow admins to freely change this field for any tool type (DE users are restricted from setting this flag in private tools).

Reply:
Ah, good catch.
Do we want these to be in ToolAdminPresenter or do we want these to happen directly in the form while the admins are changing things?

Reply:
I think we can hide the flags from the UI and let the Presenter manage them. We can add them back to the form later, if admins need to manually set these flags for some reason in the future.

The `network_mode` setting can stay in the form, though.

Reply:
One more question - ManageToolsPresenter also adds this stuff automatically.  Do we want that here as well?
https://github.com/cyverse-de/ui/blob/0cd2f34f508340b9afe79358d04140e8fc7f4983/de-lib/src/main/java/org/iplantc/de/client/models/tool/ToolAutoBeanFactory.java#L62-L69

Reply:
It looks like it already does: https://github.com/cyverse-de/ui/blob/a8b15fe363e8ece4b32e798d0d7576cd53f9de0b/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/toolAdmin/presenter/ToolAdminPresenterImpl.java#L224-L228

Reply:
Ah, you're right.  Was looking in the wrong spot. Cool, I'll push out an update